### PR TITLE
Modify the Configuration group validation cannot configure third-part…

### PR DESCRIPTION
**Pull request by zsl1549**
1、Configuration group check
2、Third party components are not configured
2、Cancel valid component pop-up update box prompt

### CODE REVIEW

File: src/pages/Configuration/Details/index.js [hunk 1]
```diff
@@ -216,12 +237,13 @@ export default class ConfigurationDetails extends PureComponent {
   checkConfiguration = (rule, value, callback) => {
     if (value && value.length > 0) {
       const arr = value.filter(
-        (item) => item.item_key === '' || item.item_value === ''
+        (item) =>
+          item.item_key === '' || (item.item_key === '' && item.item_value)
```
Comment (barnettZQG) [inline src/pages/Configuration/Details/index.js:241]: The second condition is redundant.

File: src/pages/Configuration/Details/index.js [hunk 2]
```diff
@@ -115,10 +117,23 @@ export default class ConfigurationDetails extends PureComponent {
         },
         callback: (res) => {
           if (res) {
+            const arr = [];
+            let UpDataList = [];
+            if (info && info.services && info.services.length > 0) {
```
Comment (barnettZQG) [inline src/pages/Configuration/Details/index.js:122]: What logic?

File: src/pages/Configuration/Details/index.js (hunk 1 above)
Comment (zsl1549) [reply src/pages/Configuration/Details/index.js:241]: Have to repair

File: src/pages/Configuration/Details/index.js (hunk 2 above)
Comment (zsl1549) [reply src/pages/Configuration/Details/index.js:122]: Determines whether component changes take effect

Comment (barnettZQG) [reply src/pages/Configuration/Details/index.js:126]: Components that should be updated should take a combination of the following two situations:
1. The configuration item has changed, and the effective component should be updated.
2. The component in effect has changed, and the changed component should be updated.

Comment (zsl1549) [reply src/pages/Configuration/Details/index.js:126]: It was modified in this way

**Review by barnettZQG — APPROVED** [review]

**Pull request merged**